release-tools/src/main/java/org/springframework/data/release/git/GitOperations.java

```diff
@@ -709,12 +690,12 @@ public void commit(Project project, Ticket ticket, String summary, Optional<Stri
 					.setAllowEmpty(allowEmpty) //
 					.setAll(all);
 
-			if (gpg.isGpgAvailable()) {
-				commitCommand.setSign(true).setSigningKey(gpg.getKeyname())
-						.setCredentialsProvider(new GpgPassphraseProvider(gpg));
-			} else {
-				commitCommand.setSign(false);
-			}
+			// if (gpg.isGpgAvailable()) {
+			// commitCommand.setSign(true).setSigningKey(gpg.getKeyname())
+			// .setCredentialsProvider(new GpgPassphraseProvider(gpg));
+			// } else {
+			commitCommand.setSign(false);
```

Comment:
We want to have signed commits, therefore we were using the `CredentialsProvider`.

Reply:
Fixed.